modules/backend/layouts/_head.htm

```diff
@@ -17,6 +17,7 @@
 <link href="<?= Url::asset('modules/system/assets/ui/storm.css') ?>?v<?= $coreBuild ?>" rel="stylesheet">
 <link href="<?= Backend::skinAsset('assets/css/october.css') ?>?v<?= $coreBuild ?>" rel="stylesheet">
 <script src="<?= Backend::skinAsset('assets/js/vendor/jquery.min.js') ?>?v<?= $coreBuild ?>"></script>
+<script src="<?= Backend::skinAsset('assets/js/vendor/jquery-migrate.min.js') ?>?v<?= $coreBuild ?>"></script>
```

Comment:
Doesn't this load it twice when loading it here and in october.js (compiled in october.min.js)?

Reply:
I wasn't sure if I had to add it to both? Probably best to not add it to october.js

- Just add it to only _head.htm and auth.htm

Reply:
@LukeTowers Hi it only loads once in my test October CMS see my dev tools screen shot.

![backend](https://user-images.githubusercontent.com/17784082/32724512-7bb5fb62-c869-11e7-92a2-8ee63fe3d9f5.png)